syntax = "proto3";

package zorro_python;

// Input Parameters for the concat_str rpc
message ConcatStrInput {
  // The prefix string
  string stringA = 1;
  // The suffix string
  string stringB = 2;
}

// Output Parameters for the log rpc
message ConcatStrOutput {
  // The concatenated string
  string string = 1;
}

// Simple log message
service ConcatStr {
  rpc Execute(ConcatStrInput) returns (stream ConcatStrOutput);
  rpc Undo(ConcatStrInput) returns (stream ConcatStrOutput);
}

